doc/server_API.md

```diff
@@ -78,7 +72,7 @@ On success:
 
```

Comment:
Add a code for situations when:

- There's no module with that id in our database
- There's module with that id, but it is already bound to another user